gluster-exporter/metric_pv_lv_vg_test.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+import (
+	"testing"
+)
+
+func TestMetric_PV_LV_VG(t *testing.T) {
+	t.Log("Testing started")
+	var cd collectDetails = collectDetails{cmdStr: "pvs --noheading"}
```

Comment:
Instead of calling using 3 commands for pv, lv and vg count you can do it in one command `lvm vgs --noheading --reportformat=json -o vg_name, lv_count, pv_count` . U can declare a structure say:
```
type VGCount struct {
        VGName     `json:"vg_name"`
        LVCount    `json:"lv_count"`
        PVCount    `json:"pv_count"`
}
```
and parse the json output from the command into an array of VGCount structures say VGCounts just like in https://github.com/gluster/gluster-prometheus/blob/e714bb356000d27c5fa048dd8f6acdf364f3e050/gluster-exporter/metric_brick.go#L368-L382. Now to find the vg_count you can just find len(VGCounts).